Dockerfile

```diff
@@ -68,22 +64,18 @@ RUN apk add --no-cache \
     python3 \
     python3-dev \
     py3-pip \
-    shellcheck
+    shellcheck \
+    libc6-compat
 
 # Install pre-commit framework
 RUN pip3 install --ignore-installed pre-commit==$PRECOMMIT_VERSION
 
 # Install checkov
 RUN pip3 install --ignore-installed checkov==$CHECKOV_VERSION
 
-# Download Terraform, verify checksum and install to bin dir
-RUN wget \
-    $TERRAFORM_URL \
-    $TERRAFORM_CHECKSUM_URL && \
-    sed -i '/terraform_.*_linux_amd64.zip/!d' $TERRAFORM_CHECKSUM && \
-    sha256sum -cs $TERRAFORM_CHECKSUM && \
-    unzip $TERRAFORM_ARCHIVE -d /usr/local/bin && \
-    rm -f $TERRAFORM_ARCHIVE $TERRAFORM_CHECKSUM
+# Install tfswitch and the Terraform version that is defined in the $TF_VERSION build arg
+RUN wget -O- -nv https://raw.githubusercontent.com/warrensbox/terraform-switcher/release/install.sh | sh
+RUN tfswitch $TF_VERSION
```

Comment:
@mariux this will always pre-install the Terraform version that is being set in the `TF_VERSION` **build arg.** Please note that this differs from when the environment variable `TF_VERSION` that may be set on container startup. If the env variable is set in container startup, tfswitch will automatically install and switch to the specified version. So what this means:

- The images as a certain version of Terraform pre-installed in the Image, this can be set through the build arg `TF_VERSION` in the Makefile
- To switch to another version on container startup, you may set `TF_VERSION` on container startup